base/tks/src/org/dogtagpki/server/tks/servlet/TokenServlet.java

```diff
@@ -3004,6 +3009,7 @@ private void processComputeSessionKeysSCP03(HttpServletRequest req, HttpServletR
                 sb.append("&" + IRemoteRequest.TKS_RESPONSE_EncSessionKey + "=");
                 sb.append(encSessionKeyString);
                 sb.append("&" + IRemoteRequest.TKS_RESPONSE_KekSessionKey + "=");
+                sb.append(kekSessionKeyString);
```

Comment:
interesting how this missing assignment was not found earlier.

Reply:
OH that's in the non server side keygen clause, but good to have that there for completeness.